ang/civicase/ActivityFeed.html

```diff
@@ -40,7 +40,7 @@
                 aria-labelledby="Select/Unselect"
                 ng-click="toggleSelected(activity); $event.stopPropagation();">
                 <i
-                  ng-if="selectedActivities.indexOf(activity.id) !== -1"
+                  ng-if="findActivityById(selectedActivities, activity.id)"
```

Comment:
`findActivityById` is used in many places, so naming the current name is kept generic so that it is applicable in all places.
https://github.com/compucorp/uk.co.compucorp.civicase/blob/998f8da36cd01f9c9be76d8fd278f8d5d062443e/ang/civicase/ActivityFeed.js#L60
https://github.com/compucorp/uk.co.compucorp.civicase/blob/998f8da36cd01f9c9be76d8fd278f8d5d062443e/ang/civicase/ActivityFeed.js#L161

Also this function searches on different arrays, 
https://github.com/compucorp/uk.co.compucorp.civicase/blob/998f8da36cd01f9c9be76d8fd278f8d5d062443e/ang/civicase/ActivityFeed.js#L161
vs 
https://github.com/compucorp/uk.co.compucorp.civicase/blob/998f8da36cd01f9c9be76d8fd278f8d5d062443e/ang/civicase/ActivityFeed.js#L164